To-Do, new topics, and edit topics. (#3292)
github/explore: topics/client/index.md

---

> display_name: client

Reviewer: ```suggestion
display_name: Client
```

short_description: Any computer hardware or software device that requests access to a service provided by a server.
topic: client
wikipedia_url: https://en.wikipedia.org/wiki/Client_(computing)
---

> Any computer hardware or software device that requests access to a service provided by a server. Clients are typically seen as the requesting program or user in a client-server architecture.

Reviewer: ```suggestion
A client is a piece of computer hardware or software that requests access to a service provided by a server. Clients are typically seen as the requesting program or user in a client-server architecture.
```

Reviewer: I think this would be better
